src/walletSdk/Customer/index.ts

```diff
@@ -0,0 +1,141 @@
+import { AxiosInstance } from "axios";
+import queryString from "query-string";
+import { Sep9InfoRequiredError, CustomerNotFoundError } from "../Exceptions";
+import {
+  CustomerInfoMap,
+  Sep12Status,
+  Sep12Type,
+  Field,
+  ProvidedField,
+  GetCustomerParams,
+  GetCustomerResponse,
+  AddCustomerResponse,
+} from "../Types";
+
+// Used for identifying binary fields.
+// https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0009.md
+const Sep9BinaryFields = [
```

Comment:
@acharb so we have those fields suggested by Sep-9 but Anchor and wallets could add any other "random" fields they would like to. This means if they decide on sending let's say a `photo_self` they wouldn't be able to send it as a `binary` if we limit it to what's only included in `Sep9BinaryFields`.

A more generic/flexible approach would be augmenting the `CustomerInfoMap` to account for field types.

Something like:
```
export type CustomerInfoMap = {
  [field_key: string]: {
     type: Sep12Type;
     value: any;
   }
};
```

So we could detect if we have any `binary` field being passed to set the "multipart/form-data" and append those `binary` fields as the last fields in the request body.

What do you think?

Reply:
hummm maybe a much simpler approach would be just adding a `binaries` array to the add/update function input in which the client could list the field keys which are `binary`.

Reply:
I like the idea of adding a binary field, but I was thinking the binary map similar to the sep9Info.

I like this idea as well so that the user doesn't have to remember that binary fields need to be in the map last

lmk what you think

https://github.com/stellar/typescript-wallet-sdk/pull/71/commits/263cbaa3b6740ca0bc6c691aa980843dc70d6990

Reply:
love it!

Reply:
thanks for the help  🙌